synapse/models/crypto.py

```diff
@@ -42,9 +43,30 @@ def getModelDefs(self):
                     'doc': 'A hex encoded Microsoft Windows NTLM password hash',
                     'ex': ex_md5
                 }),
+
                 ('rsa:key', ('comp', {'fields': (('mod', 'hex'), ('pub:exp', 'int')), }), {
                     'doc': 'An RSA keypair modulus and public exponent.'
                 }),
+
+                ('crypto:x509:cert', ('guid', {}), {
+                    'doc': 'A unique X.509 certificate.',
+                }),
+
+                ('crypto:x509:san', ('comp', {'fields': (('type', 'str'), ('value', 'str'))}), {
+                    'doc': 'An X.509 Subject Alternative Name (SAN).',
+                }),
+
+                ('crypto:x509:crl', ('guid', {}), {
```

Comment:
CRLs are also GeneralNames like SANs, should they have the same (type, value) fields?